ui/src/map/plan-download/plan-download-plan-section.tsx

```diff
@@ -4,28 +4,65 @@ import styles from './plan-download-popup.scss';
 import { Button, ButtonSize, ButtonVariant } from 'vayla-design-lib/button/button';
 import { Checkbox } from 'vayla-design-lib/checkbox/checkbox';
 import { IconColor, Icons, IconSize } from 'vayla-design-lib/icon/Icon';
-import { GeometryPlanId, PlanApplicability } from 'geometry/geometry-model';
+import {
+    GeometryPlanId,
+    highestApplicability,
+    PlanApplicability,
+    PlanSource,
+} from 'geometry/geometry-model';
 import { DownloadablePlan } from 'map/plan-download/plan-download-store';
+import { inframodelBatchDownloadUri } from 'infra-model/infra-model-api';
+import { createClassName } from 'vayla-design-lib/utils';
+import { LayoutTrackNumberId, LocationTrackId } from 'track-layout/track-layout-model';
+import { KmNumber } from 'common/common-model';
 
 type PlanItemProps = {
     id: GeometryPlanId;
     name: string;
     checked: boolean;
     applicability: PlanApplicability | undefined;
+    source: PlanSource;
     setPlanSelected: (planId: GeometryPlanId, selected: boolean) => void;
+    selectPlan: (planId: GeometryPlanId) => void;
```

Comment:
Note to self: Parempi erotus näiden nimille. `selectPlan` käskyttää meidän `Selection`:ia, `setPlanSelected` merkkaa suunnitelman ladattavaksi